### ISSUE TITLE
IndexError: list index out of range Company Scraper

### ISSUE THREAD

**Issue by mohitag09** [joeyism/linkedin_scraper#54]
Following error by using ver. 2.5.2 as well. Now its not giving the exact line as well where the error is happening
```
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-10-1fef8da42f7f> in <module>
    ---> 12 company = Company(linkedin_url = "https://www.linkedin.com/company/google",driver=driver,close_on_complete=False,get_employees=False)
     13 set_trace()
     14 print(company)

~\AppData\Roaming\Python\Python38\site-packages\linkedin_scraper\company.py in __init__(self, linkedin_url, name, about_us, website, headquarters, founded, company_type, company_size, specialties, showcase_pages, affiliated_companies, driver, scrape, get_employees, close_on_complete)

~\AppData\Roaming\Python\Python38\site-packages\linkedin_scraper\company.py in scrape(self, get_employees, close_on_complete)

~\AppData\Roaming\Python\Python38\site-packages\linkedin_scraper\company.py in scrape_logged_in(self, get_employees, close_on_complete)

IndexError: list index out of range
```

**Comment by joeyism** [comment]
Does it only fail for you for python 38? It works for me locally for python 36

**Comment by lxuis97** [comment]
Hi do have this, following the update for my part: 

Traceback (most recent call last):
  File "linkedin_search.py", line 54, in <module>
    lk_get_company_info('https://www.linkedin.com/company/sicara/')
  File "linkedin_search.py", line 32, in lk_get_company_info
    company.scrape(close_on_complete=False, get_employees=False)
  File "/Users/Louis/Library/Python/3.8/lib/python/site-packages/linkedin_scraper/company.py", line 79, in scrape
    self.scrape_logged_in(get_employees = get_employees, close_on_complete = close_on_complete)
  File "/Users/Louis/Library/Python/3.8/lib/python/site-packages/linkedin_scraper/company.py", line 150, in scrape_logged_in
    self.__find_first_available_element__(
AttributeError: 'NoneType' object has no attribute 'click'

**Comment by joeyism** [comment]
Fixed as of [2.5.4](https://github.com/joeyism/linkedin_scraper/releases/tag/2.5.4)
